internal/clients/team5/Speaker.go

```diff
@@ -3,13 +3,55 @@ package team5
 import (
 	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 )
 
 type speaker struct {
 	*baseclient.BaseSpeaker
+	c *client
 }
 
 func (c *client) GetClientSpeakerPointer() roles.Speaker {
 	c.Logf("Team 5 became speaker")
 	return &c.team5Speaker
 }
+
+// Pay Judge based on the status of our own wealth
+// If we are not doing verywell, pay Judge less so we have more in the CP to take from
+func (s *speaker) PayJudge() shared.SpeakerReturnContent {
+	JudgeSalaryRule, ok := rules.RulesInPlay["salary_cycle_judge"]
```

Comment:
I think they've changed `rules` package.. Saw that `rules.RulesInPlay` is no longer defined. See how it was changed for president: `SpeakerSalaryRule, ok := pres.GameState.RulesInfo.CurrentRulesInPlay["salary_cycle_speaker"]
`